graveler/committed/batch_test.go

```diff
@@ -126,18 +126,19 @@ func TestBatchCloserMultipleWaitCalls(t *testing.T) {
 		Count:   4321,
 	}, nil)
 
-	sut := runSuccessScenario(t)
+	sut := runSuccessScenario(t, 1, 1)
 
 	assert.Error(t, sut.CloseWriterAsync(writer), committed.ErrMultipleWaitCalls)
 	res, err := sut.Wait()
 	require.Nil(t, res)
 	require.Error(t, err, committed.ErrMultipleWaitCalls)
 }
 
-func runSuccessScenario(t *testing.T) *committed.BatchCloser {
-	const writersCount = 10
-	writers := make([]*FakeRangeWriter, writersCount)
-	for i := 0; i < writersCount; i++ {
+func runSuccessScenario(t *testing.T, writersAmount, closersAmount int) *committed.BatchCloser {
```

Comment:
```suggestion
func runSuccessScenario(t *testing.T, numWriters, numClosers int) *committed.BatchCloser {
```
Rationale: "amount" is for uncountable quantities, these parameters specifically count.